compiler/codegen/cg-expr.cpp

```diff
@@ -3831,13 +3855,20 @@ void codegenAssign(GenRet to_ptr, GenRet from)
           // Make sure that from is a pointer
           codegenCopy(to_ptr, from, type);
         } else {
-          codegenCall("chpl_gen_comm_put",
-                      codegenCastToVoidStar(codegenValuePtr(from)),
-                      codegenRnode(to_ptr),
-                      codegenRaddr(to_ptr),
-                      codegenSizeof(type),
-                      genCommID(info),
-                      info->lineno, gFilenameLookupCache[info->filename]);
+          std::vector<GenRet> args;
```

Comment:
I agree, but I want to pass since also you don't seem to be pushing hard.

Handling that call to `codegenValuePtr` will make the helper unnecessarily hard to read in my view. But "wait till the third time" is a strategy I can get behind.